app/models/pageflow/draft_entry.rb

```diff
@@ -28,6 +29,11 @@ def initialize(entry, draft = nil)
       @draft = draft || entry.draft
     end
 
+    # So we can always get to the original Entry title.
+    def entry_title
+      entry.title
+    end
+
```

Comment:
This collides with the delegation to revision. This should probably behave like PublishedEntry. Maybe the test for DraftEntry can be extracted to a shared example group and be reused?